pkg/controller/node/nodecontroller.go

```diff
@@ -505,7 +505,7 @@ func (nc *NodeController) onNodeDelete(originalObj interface{}) {
 			glog.Errorf("Received unexpected object: %v", obj)
 			return
 		}
-		node, ok = deletedState.Obj.(*v1.Node)
+		_, ok = deletedState.Obj.(*v1.Node)
```

Comment:
@wojtek-t is right - 508 is not defining new Node object, but sets the value of one defined in 499.